package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@klarna/ui-css-components",
-  "version": "5.6.4",
+  "version": "5.7.0",
```

Comment:
Agreed. I updated the PR and I'm merging as 5.6.5
Also - we should all be aligned about using ```npm version``` to manage these updates for us so we don't miss out on things like tagging